litellm/llms/vertex_ai.py

```diff
@@ -328,11 +328,9 @@ def completion(
         # vertexai does not use an API key, it looks for credentials.json in the environment
 
         prompt = " ".join(
-            [
-                message["content"]
-                for message in messages
-                if isinstance(message["content"], str)
-            ]
+            str(item)
+            for message in messages
+            for item in (message["content"] if isinstance(message["content"], (str, list)) else [message["content"]])
```

Comment:
my preference would be to have a consistent way of assuming what each message would look like. so we should probably be handling this translation upstream.

Reply:
i agree if you guys can reproduce the issue and now know where the it is that's all i need from the pr